src/signals/settings/roles/containers/RoleFormContainer/index.js

```diff
@@ -32,37 +38,54 @@ export const RoleFormContainer = ({
   },
   onPatchRole,
   onSaveRole,
+  showGlobalNotification,
+  onResetResponse,
   userCan,
 }) => {
   const { roleId } = useParams();
+  const location = useLocation();
+  const history = useHistory();
   const role = list.find(item => item.id === roleId * 1);
   const title = `Rol ${roleId ? 'wijzigen' : 'toevoegen'}`;
+  const redirectURL = {
+    pathname: location.referrer || routes.roles,
+    state: location.state,
```

Comment:
I don't think that `state` is used on the overview page, or is it?

Reply:
weg